tests/src/Interop/COM/NativeServer/NumericTesting.h

```diff
@@ -8,6 +8,8 @@
 #include <limits>
 #include "Servers.h"
 
+#define THROW_FAIL_IF_FALSE(exp) { if (!(exp)) { ::printf("FALSE: %s\n", #exp); throw E_FAIL; } }
```

Comment:
Remove? I don't think this is needed/used.

Reply:
Yep, that's a change that I decided I didn't need.